access-control-app/src/cmr/access_control/data/access_control_index.clj

```diff
@@ -25,27 +25,27 @@
 
 (defmapping ^:private group-mappings group-type-name
   "Defines the field mappings and type options for indexing groups in elasticsearch."
-  {:concept-id (m/stored m/string-field-mapping)
```

Comment:
Just curious: why get rid of `m/stored`?

Reply:
By enabling `source`, all fields are stored in `_source` by default and we don't need to explicitly store any fields anymore.